lms/djangoapps/grades/tests/test_new.py

```diff
@@ -459,12 +459,6 @@ def setUp(self):
         self.request = get_mock_request(UserFactory())
         self.client.login(username=self.request.user.username, password="test")
         CourseEnrollment.enroll(self.request.user, self.course.id)
-        course_structure = get_course_blocks(self.request.user, self.course.location)
```

Comment:
**Note to reviewers:** This change was needed since the `test_graded_metadata_alterations` and `test_weight_metadata_alteration` tests were not updating course_structure after the course was updated.  This issue was surfaced because of the removal of the defensive code in block_structure.py.  Before, the code was acting on a NULL block and didn't realize it.